Firestore/core/src/firebase/firestore/core/query_listener.h

```diff
@@ -83,9 +83,9 @@ class QueryListener {
     return snapshot_;
   }
 
-  virtual void OnViewSnapshot(ViewSnapshot snapshot);
+  virtual bool OnViewSnapshot(ViewSnapshot snapshot);
```

Comment:
`bool` and other primitive return values or argument types pretty much always need a comment that explains what they main. Please add comments to these declarations.